ChegeBryan/politico
# #163545907 Admin can edit party

#### What does this PR do?

Build page for admin, enabling edit political parties
#### Description of Task to be completed?

- [x] Create button to trigger edit action
- [x] Create a form to enter new edit details 

#### How should this be manually tested?
clone this at `https://github.com/ChegeBryan/politico.git` and check out to `ft-admin-dashboard-163545907` jump in the `UI` directory and open `edit_party.html` on your favourite browser.

#### What are the relevant pivotal tracker stories?
[#163545907](https://www.pivotaltracker.com/story/show/163545907)

#### Screenshots
##### table with edit action
![screencaptuui-edit_party-html-2019-01-31-21_00_41](https://user-images.githubusercontent.com/40359451/52074988-3670d180-259c-11e9-8b8a-ad41ffeacba7.png)
##### edit form
![screencapture-1-ui-edit_party-html-2019-01-31-21_01_08](https://user-images.githubusercontent.com/40359451/52075038-4f798280-259c-11e9-8ebd-7edd8393bfbc.png)
